src/olympia/ratings/admin.py

```diff
@@ -52,11 +54,20 @@ class RatingAdmin(admin.ModelAdmin):
               'user_link', 'deleted')
     list_display = ('id', 'addon', 'created', 'user', 'ip_address', 'rating',
                     'is_reply', 'flag', 'deleted', 'truncated_body',)
-    list_filter = ('deleted', RatingTypeFilter)
+    list_filter = ('deleted', RatingTypeFilter, 'rating')
     actions = ('delete_selected',)
+    list_select_related = ('user',)  # For addon/reply_to see get_queryset()
```

Comment:
This is what fixes #11333 ; the rest of the queryset improvements are just to lower the number of queries.